### PR TITLE
Graph schema

### PR DESCRIPTION
**Pull request by miteshvp**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -211,6 +211,19 @@ services:
...
+      <<: *worker_environment
+      WORKER_ADMINISTRATION_REGION: ingestion
+    restart: on-failure
+    container_name: coreapi-worker-graph-schema
+    network_mode: bridge
+    links:
+     - gremlin-http
+    command: python3 /graph/create_graph_schema.py
+    depends_on:
+     - gremlin-http
```
Comment (fridex) [inline docker-compose.yml]: `network_mode` can be omitted.

File: docker-compose.yml [hunk 2]
```diff
@@ -200,6 +198,8 @@ services:
       GCP_AUTH_PROVIDER_X509_CERT_URL: ""
       GCP_CLIENT_X509_CERT_URL: ""
       SCANCODE_PROCESSES: "4"
+      AWS_S3_ACCESS_KEY_ID: GNV3SAHAHA3DOT99GQII
+      AWS_S3_SECRET_ACCESS_KEY: ZmvMwngonaDK5ymlCd6ptaalDdJsCn3aSSxASPaZ
```
[the remark this answers is not in the document]
Comment (miteshvp) [reply docker-compose.yml]: fixed. Thanks

File: docker-compose.yml (hunk 1 above)
Comment (miteshvp) [reply docker-compose.yml]: fixed. Thanks

**Comment by tisnik** [comment]
[test this]

**Comment by tisnik** [comment]
@miteshvp is this PR still relevant? If not, we might close it.

**Comment by miteshvp** [comment]
Not relevant anymore.

**Pull request closed without merging**